### ISSUE TITLE
Specify `[alt]` on `<img>`

### ISSUE THREAD

**Issue by wooorm** [larsdouweschuitema/minor-meesterproef#40]
Always add an `[alt]` attribute on `<img>` element, but use an empty value if there’s no semantic text that should be read aloud to screen-reader users.

[`views/head.jade#L11`](https://github.com/larsdouweschuitema/minor-meesterproef/blob/ef61972bedec3a724d782060406bc9f95f60f97b/views/head.jade#L11):

```diff
 a(href='/') Logo
-img(src='/img/menu.svg')
+img(src='/img/menu.svg', alt='')
 input(type='text')
```

**Comment by larsdouweschuitema** [comment]
Yes I left it out due to the screen reader. Why would I specify an empty alt tag?
